public/style.css

```diff
@@ -184,19 +206,46 @@ a {
 }
 
 .numberCircle {
+
```

Comment:
These circles feel way too large, I'd size them down to be about 32px in size with a 16px font-size for the text.

Reply:
fixed